W-16437507 | AM Agent not supported by Upgrade Tool (#2777)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/mule-upgrade-tool.adoc

If the Runtime Manager agent is installed in your current Mule instance, the upgrade tool also updates the agent version as part of the upgrade process.

> Upgrading Anypoint Monitoring Agent is not supported, in order to proceed with an upgrade in cases where that Agent is installed, it is needed to uninstall the Agent prior to upgrade, and reinstall it after upgrading. See xref:monitoring::am-installing.adoc#update-the-anypoint-monitoring-agent[Update the Anypoint Monitoring Agent].

Reviewer: ```suggestion
Upgrading the Anypoint Monitoring agent isn't supported. To proceed with an upgrade, you must first uninstall the agent, and reinstall it after the Mule instance upgrade is complete. See xref:monitoring::am-installing.adoc#update-the-anypoint-monitoring-agent[Update the Anypoint Monitoring Agent].
```
